src/logger/loggerApi.c

```diff
@@ -1470,7 +1471,7 @@ int api_getScript(Serial *serial, const jsmntok_t *json){
 int api_setScript(Serial *serial, const jsmntok_t *json){
 
 	int returnStatus = API_ERROR_UNSPECIFIED;
-
+	bool should_reload_script = false;
```

Comment:
[Pedantic] Could shorten this variable name.  See Chapter 4 in https://www.kernel.org/doc/Documentation/CodingStyle